presentation/src/main/java/com/velogm/presentation/ui/home/screenhome/ScreenHomeSlidePageFragment.kt

```diff
@@ -0,0 +1,70 @@
+package com.velogm.presentation.ui.home.screenhome
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.viewModels
+import androidx.lifecycle.flowWithLifecycle
+import androidx.lifecycle.lifecycleScope
+import com.velogm.core_ui.base.BindingFragment
+import com.velogm.core_ui.view.UiState
+import com.velogm.presentation.R
+import com.velogm.presentation.databinding.ItemFragmentHomeBinding
+import dagger.hilt.android.AndroidEntryPoint
+import kotlinx.coroutines.flow.launchIn
+import kotlinx.coroutines.flow.onEach
+import timber.log.Timber
+
+@AndroidEntryPoint
+class ScreenHomeSlidePageFragment :
+    BindingFragment<ItemFragmentHomeBinding>(R.layout.item_fragment_home_) {
+
+    private lateinit var postAdapter: PostAdapter
+    private val viewModel by viewModels<ScreenViewModel>()
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        postAdapter = PostAdapter(deleteTagClick = {
+            Timber.tag("Post").d(it.url)
+        })
+        binding.rvPostList.adapter = postAdapter
+
+        val data = arguments?.getString("data")
+
+        if (data == "트렌드") {
+            viewModel.getTrendPost()
+            Timber.d("11111111")
+        }
+        if (data == "팔로우") {
+            viewModel.getFollowPost()
+            Timber.d("22222222")
+        }
+        if (data != "트렌드" && data != "팔로우") {
+            viewModel.getTagPost(data.toString())
+            Timber.d("33333333")
+        }
```

Comment:
```suggestion
          when(data){
            "트렌드" -> viewModel.getTrendPost()
            "팔로우" -> viewModel.getFollowPost()
            else -> viewModel.getTagPost(data.toString())
        }

```
이거 이렇게 바꿔보는 건 어떠신지,,

Reply:
아 저기 원래 저렇게 했다가 테스트 할게 있어서요 아예 저부분 사라질수도!!